reporting/api/src/main/java/org/jboss/windup/reporting/model/JarDependenciesReportModel.java

```diff
@@ -0,0 +1,33 @@
+package org.jboss.windup.reporting.model;
+
+import org.jboss.windup.graph.model.ProjectModel;
+
+import com.tinkerpop.blueprints.Direction;
+import com.tinkerpop.frames.Incidence;
+import com.tinkerpop.frames.modules.typedgraph.TypeValue;
+
+/**
+ * This represents Jar Dependencies report model
+ * 
+ * @author mnovotny
+ *
+ */
+@TypeValue(JarDependenciesReportModel.TYPE)
+public interface JarDependenciesReportModel extends ApplicationReportModel
+{
+    String TYPE = "jarDependenciesReport";
+
+    String JAR_REPORT_TO_PROJECT_MODEL = "jarDepsToProjectModel"; 
+
+    /**
+     * Contains all {@link ProjectModel}s that contain this file
+     */
+    @Incidence(label = JAR_REPORT_TO_PROJECT_MODEL, direction = Direction.OUT)
```

Comment:
I do not think it will necessarily need a JavaHandler method. Using some specific names for the relationships, it might look something like this:

JarDependencyReportModel -> DependencyModel (SHA1:String) -> DependencyToArchiveEdgeModel (path:String) -> ArchiveModel

Feel free to use better names if you think of them.

It might also make sense to maintain a direct link from DependencyModel to the canonical project to make displaying the information on the report easier.